automated install/basic-install.sh

```diff
@@ -90,7 +90,6 @@ IPV6_ADDRESS=${IPV6_ADDRESS}
 QUERY_LOGGING=true
 WEBPORT=8080
 PRIVACY_LEVEL=0
-CACHE_SIZE=10000
```

Comment:
I removed this because of a `shellcheck` warning. It an abandoned variable not used anywhere in the `core` code but managed by FTL alone in v6